rotkehlchen/serialization/deserialize.py

```diff
@@ -588,7 +589,22 @@ def deserialize_evm_transaction(
             ), None
 
         # else normal transaction
-        gas_price = read_integer(data=data, key='gasPrice', api=source)
+        gas_price = read_integer(data=data, key='gasPrice', api=source)  # gas price bid
+
+        if chain_id == ChainID.ARBITRUM_ONE:
+            if block_data is None:
```

Comment:
This makes no sense as I told you in discord.

1. It can't be that you need more data during deserialization force at least 1 external call per arbitrum transaction.
2. This will never work. `evm_inquirer` is actually not passed during transaction query. https://github.com/rotki/rotki/blob/11ca018f30f277e86460429d9bbc964c02aaa31b/rotkehlchen/externalapis/etherscan.py#L383 and https://github.com/rotki/rotki/blob/11ca018f30f277e86460429d9bbc964c02aaa31b/rotkehlchen/externalapis/etherscan.py#L402.
3. Have you tested it with a normal node? This would only pass the inquirer then. But are the keys of the transaction the same there?